pandas/tests/arithmetic/test_timedelta64.py

```diff
@@ -313,7 +313,9 @@ def test_subtraction_ops(self):
         tm.assert_index_equal(result, expected, check_names=False)
 
         result = dti - td
-        expected = DatetimeIndex(["20121231", "20130101", "20130102"], name="bar")
+        expected = DatetimeIndex(
+            ["20121231", "20130101", "20130102"], freq="D", name="bar"
```

Comment:
Is this a bugfix with the `freq`? Worth a whatsnew?

Reply:
No.  ATM assert_index_equal doesnt check for matching `freq`.  Moving towards adding that check, but ATM it breaks about 500 tests.  Honestly not sure why this edit made it into this branch.